src/components/HikeSearch/index.js

```diff
@@ -23,11 +23,15 @@ class HikeSearch extends Component {
     const searchTerm = keyword.trim().replace(/\\|\/|  +/g, ' ');
     if (searchTerm.length > 2) {
       const URL = !isDev ? BASE_PATH_URL : '';
-      const searchUrl = `${URL}/hikes/search?keyword=${searchTerm}`;
+      const params = new URLSearchParams({
+        ...Router.params,
+        keyword: searchTerm,
```

Comment:
Minor suggestion: The , at the end of "searchTerm" can be omitted.

Reply:
This repo has eslint enabled with airbnb's rules, which enforces trailing commas:
https://github.com/airbnb/javascript#commas
https://eslint.org/docs/rules/comma-dangle